backend/apps/mentorship/Makefile

```
@@ -0,0 +1,2 @@
+filter/link:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)